client/src/main/scala/drt/client/components/TerminalDashboardComponent.scala

```diff
@@ -149,8 +150,8 @@ object TerminalDashboardComponent {
             ))(^.className := "terminal-dashboard-side__sidebar_widget", "View Arrivals"),
           <.div(
             ^.className := "terminal-dashboard-side__sidebar_widget time-slot-changer",
-            <.label(^.className := "terminal-dashboard-side__sidebar_widget__label", "Time slot duration"),
-            <.select(
+            <.label(^.className := "terminal-dashboard-side__sidebar_widget__label", ^.aria.label := "Select Time duration slot to show pcp passenger on Dashboard", "Select Time slot duration"),
```

Comment:
`Select time slot`? I'm not sure we need to say 'to show pcp passengers on dashboard'

Reply:
It is for screen reader so select time slot for what purpose is not clear to I mentioned to `show pcp passengers on Dashboard`. 
Maybe we can say . `Select time slot to display pcp passenger numbers on the Dashboard accordingly to timeslot`